daemon/api.go

```diff
@@ -900,3 +906,7 @@ func appIconGet(c *Command, r *http.Request) Response {
 
 	return FileResponse(path)
 }
+
+func getCapabilities(c *Command, r *http.Request) Response {
+	return SyncResponse(c.d.capRepo.All())
```

Comment:
One more detail here: by returning a list you've closed the response down to any further improvements. For example, once we need to paginate, or add any further metadata to the response, we'll need to break compatibility. I'd suggest adding the list inside a "capabilities" key, similar to how packages work (in fact, the result should likely be a map of name=>cap, as in packages). This will allow you to evolve forward without breaking backwards.